DSP Accel Library/slblocks.m

```diff
@@ -0,0 +1,13 @@
+function blkStruct = slblocks
+		% This function specifies that the library should appear
+		% in the Library Browser
+		% and be cached in the browser repository
+
+		Browser.Library = 'DSP_FPGA_Accelerated_Toolbox';
```

Comment:
Let's change the library name to "FPGA Open Speech Tools" or something similar to reflect our GitHub Organization name.

Reply:
We want a single library (potentially with sub-libraries like other Simulink libraries do). This library can live in the root of the repository if we want, rather than in a subdirectory. 

I'm open to suggestions on directory organization here. 

The DSP Builder library that was in this repo has been removed, so now this library can be the main one. Rebase against master, then you can put this library in the root of the repository.

Reply:
I'm not super sure on how to set up the library structure. I've done a decent amount of searching and still don't have a good understanding. How would I make this library not a sub-library?

Reply:
It's a library right now. I'm not super familiar with doing anything more than you have either. I don't know if sub-library is the correct term. An example of what I'm thinking about is, for example, the HDL Coder section has a Discrete subsection.

Reply:
> > I'll get to work on some READMEs that will point to where I've written about usage of the blocks.
> > I wrote some documentation that's hidden somewhere by the masks, I'll check on how to make that more visible, without just showing up at the top of the mask.
> 
> Sounds good. The READMEs should contain enough information for users to figure out how to use the blocks. If they want more information, we can point them to the code and mask documentation as well.

I'm rewriting and adding to the "help" context menu documentation.
Also made a huge change with a new setting "save Input/Output set as .mat" for the PLUT, where instead of having to know all the ins and outs of how xIn is calculated and exactly what each address is associated with, it just saves a .mat file with those 3 rows. Working on exactly how I want that .mat file to look, both for usability and for ease of separation/workspace readability. The idea is to make reprogramming the table much easier, but it's possible more than one of these tables will be in the workspace at once.

I'll have a README that gives brief description on how to use each block, but the "help" documentation and markdown documentation will go much more in depth on I/O and inner workings.